park-dashboard/src/Components/ParkDesignations/DesignationPolarArea.jsx

```diff
@@ -0,0 +1,77 @@
+import React, { useState, useEffect } from "react";
+import { LoadParkData } from "../../LoadParkData";
+import {
+  Chart as ChartJS,
+  RadialLinearScale,
+  ArcElement,
+  Tooltip,
+  Legend,
+} from "chart.js";
+import { PolarArea } from "react-chartjs-2";
+
+ChartJS.register(RadialLinearScale, ArcElement, Tooltip, Legend);
+
+const DesignationPolarArea = (parks) => {
+  const parkData = LoadParkData(parks);
+
+  //Fetch all the park designations
+  const parkDesignations = parkData.map((park) => park.designation);
+
+  //Unique park designations
+  const uniqueDesignations = [...new Set(parkDesignations)];
+  console.log(uniqueDesignations);
+
+  //Count of each unique designation
+  const counts = parkDesignations.reduce((acc, designation) => {
+    acc[designation] = (acc[designation] || 0) + 1;
+    return acc;
+  }, {});
+
+  const countsOfDesignations = JSON.stringify(counts);
+  console.log(countsOfDesignations);
```

Comment:
Same as above

Reply:
Thanks for noticing this, I have removed the console.log statements.